pyp2rpm/metadata_extractors.py

```diff
@@ -355,7 +363,8 @@ def data_from_archive(self):
         archive_data['python_versions'] = py_vers[1:] if py_vers \
             else [settings.DEFAULT_ADDITIONAL_VERSION]
 
-        archive_data['doc_files'] = self.doc_files
+        archive_data['doc_files'], archive_data[
+            'doc_license'] = self.separate_license_files(self.doc_files)
```

Comment:
Any chance of some more readable line break? Not necessarily, but this feel, ehm, ugly?